qtaskbarcontrol.h

```diff
@@ -22,9 +24,10 @@ class Q_TASKBAR_CONTROL_EXPORT QTaskbarControl : public QObject
 	};
 	Q_ENUM(WinProgressState)
 
-	explicit QTaskbarControl(QWidget *parent);
+	explicit QTaskbarControl(QObject *parent);
 	~QTaskbarControl() override;
 
+	void setWindow(QWidget *widget);
```

Comment:
Please add an overload for `void setWindow(QWindow *window);` that would look something like this:
```
void QTaskbarControl::setWindow(QWindow *window) {
    // remove eventfilter from current QWidget, if applicable
    d->setWindow(window);
}
```

Reply:
Yes i will add this too